tribe-ext-community-currency-symbol/index.php

```diff
@@ -0,0 +1,402 @@
+			$default_position = 'suffix';
+		} else {
+			$default_position = 'prefix';
+		}
+
+		$default_position = apply_filters( 'tribe_ext_ce_cost_currency_symbol_position_default', $default_position );
+
+		return sanitize_text_field( $default_position );
+	}
+
+	/**
+	 * Get the symbol position phrase
+	 *
+	 * @param string $position
+	 *
+	 * @return string
+	 */
+	protected function get_symbol_position_phrase( $position = 'prefix' ) {
+		if ( 'prefix' === $position ) {
+			$output = esc_attr_x( 'Before Cost', 'Currency symbol position', 'tribe-extension' );
+		} else {
+			$output = esc_attr_x( 'After Cost', 'Currency symbol position', 'tribe-extension' );
+		}
+
+		return $output;
+	}
+
+	/**
+	 * Build the list of select options for the allowed currency symbols.
+	 *
+	 * @return string
+	 */
+	protected function symbol_list_select_options() {
+		$symbol_list_array = $this->symbol_list_array();
+
+		if ( empty( $symbol_list_array ) ) {
+			return '';
+		}
+
+		$current_symbol = (string) $this->get_current_symbol();
+
+		$post_id = Tribe__Main::post_id_helper();
+
+		/**
+		 * If currency symbol is not set and we verify we are on the Add New
+		 * Event version of the Community Events form, then set the currency
+		 * symbol to the default setting. This avoids setting a purposefully-blank
+		 * currency symbol to the default symbol when on the Edit Event version of
+		 * the Community Events form.
+		 */
+		if (
+			'' === $current_symbol
+			&& $this->is_community_page( 'add-event' )
+		) {
+			$current_symbol = tribe_get_option( 'tribe_ext_ce_cost_currency_symbol_default' );
+		}
+
+		$output = sprintf( '%1$s<option value=""></option>%1$s', PHP_EOL );
+
+		foreach ( $symbol_list_array as $key => $value ) {
+			$value = esc_attr( $value );
+
+			$selected = selected( $value, $current_symbol, false );
+
+			$opt_text = esc_attr_x( $value, 'Currency symbol', 'tribe-extension' );
+
+			$output .= sprintf( '<option value="%s" %s>%s</option>%s', $value, $selected, $opt_text, PHP_EOL );
+		}
+
+		return $output;
+	}
+
+	/**
+	 * Build the list of select options for the symbol position.
+	 *
+	 * @return string
+	 */
+	protected function symbol_position_list_select_options() {
+		$default_position = $this->get_symbol_position_default();
+
+		if ( 'prefix' === $default_position ) {
+			$opposite = 'suffix';
+		} else {
+			$opposite = 'prefix';
+		}
+
+		$symbol_position_list_array[$opposite] = $this->get_symbol_position_phrase( $opposite );
+
+		$current_position = (string) $this->get_current_symbol_position();
+
+		$post_id = Tribe__Main::post_id_helper();
+
+		// Awaiting https://central.tri.be/issues/80685 instead of the empty() check here
+		if ( '' === $current_position && empty( $post_id ) ) {
+			$current_position = $default_position;
+		}
+
+		$output = sprintf( '%1$s<option value=""></option>%1$s', PHP_EOL );
+
+		foreach ( $symbol_position_list_array as $key => $value ) {
+			$key = esc_attr( $key );
+
+			$value = esc_attr( $value );
```

Comment:
looks like this doesn't need to be escaped here; it's escaped later as part of `$opt_text` declaration